pkg/githubsource/receive_adapter.go

```diff
@@ -61,16 +68,20 @@ func (ra *GitHubReceiveAdapter) handleEvent(payload interface{}, gitHubEventType
 	cloudEventType := fmt.Sprintf("%s.%s", sourcesv1alpha1.GitHubSourceEventPrefix, gitHubEventType)
 	source := sourceFromGitHubEvent(gh.Event(gitHubEventType), payload)
 
-	return ra.postMessage(payload, source, cloudEventType, eventID)
+	return ra.postMessage(payload, source, cloudEventType, eventID, hdr)
 }
 
-func (ra *GitHubReceiveAdapter) postMessage(payload interface{}, source, eventType, eventID string) error {
+func (ra *GitHubReceiveAdapter) postMessage(payload interface{}, source, eventType, eventID string, hdr http.Header) error {
```

Comment:
isn't `eventId` in the headers ?

Reply:
I made a mess in these methods, fixing it.

Reply:
Expanding on this: I think it still makes sense to propagate both GH custom headers even if X-GitHub-Delivery is also put in the cloud event id. The GH sdk will typically expect the original headers to be there. If we remove the -Delivery one, a consumer downstream would have to reinject it based on the cloud event ID, which not obvious. On the other hand, recovering all CE-X- headers blindly would immediately be SDK friendly. Does that make sense?